pkg/action/render_template.go

```diff
@@ -26,7 +26,7 @@ func RenderTemplate(path string, config *config.Config, runtime state.Runtime) (
 
 	result := new(bytes.Buffer)
 	err = tpl.Execute(result, map[string]interface{}{
-		"Config": config.Config,
+		"Config": config.Config.Values,
```

Comment:
I'm not sure about this one. The test template looks like this:

```
configTest: "{{.Config.testKey | upper}}"
stateTest: "{{.State.system.os.architecture}}"
```
but given `Config` is an object of [this type](https://github.com/kairos-io/kairos-agent/blob/a8e855b9337fe92688148a6aac2281c3acc2397d/pkg/config/config.go#L124), up to know, one could use this in a template: `{{ .Config.Bundles }}` or `{{ .Config.EjectCD }}`

By only passing down the `Config.Values` to the template, we allow them to refer only to user config values instead. I'm not sure if it was every desirable to allow them to access our own object fields.

Reply:
The alternative would be to tell the users that user config values would be accessible as `{{ .Config.Values.testKey }}` which is a bit weird because we need to document that they also have access to a completely undocumented object from code as well (which nobody knew so far?)

Reply:
well, do we want to expose in the templates internal values used for things like install and such? Do we know of the repercussions of passing a logger object or a mounter or an fs to the templating stuff?

IMO I would leave only the values in there, restrict the rest. If someone was using that then... weird.

Reply:
I agree. ok let's go with this solution here then. If someone complains, then at least we'll know what the use case is. It was not documented anyway, if anyone was (ab)using it, they did it by looking at the code.

Reply:
https://github.com/kairos-io/kairos-agent/pull/562